autospec/buildreq.py

```diff
@@ -775,6 +775,11 @@ def scan_for_configure(dirn):
         parse_r_description(os.path.join(dirn, "DESCRIPTION"))
     elif buildpattern.default_pattern == "phpize":
         add_buildreq("buildreq-php")
+    elif buildpattern.default_pattern == "nginx":
+        add_buildreq("nginx-mainline-dev")
+        add_buildreq("nginx-mainline-extras-modulebuild")
+        add_buildreq("zlib-dev")
+        add_buildreq("pcre-dev")
```

Comment:
Would you mind making a buildreq-nginx for these?

Reply:
Done, live